### PR TITLE
SAK-45964: SiteStats: Error on log when exporting to PDF a report by events to Table

### PR DESCRIPTION
**Pull request by jesusmmp**
https://jira.sakaiproject.org/browse/SAK-45964

Before:
image
image
image

After:
image
image
image

**Icons on "Export PDF" has been removed. No errors appears on the log with the patch.**

### CODE REVIEW

File: sitestats/sitestats-tool/src/java/org/sakaiproject/sitestats/tool/wicket/pages/ReportsEditPage.java [hunk 1]
```diff
@@ -913,13 +916,16 @@ public String getStyle(Object object) {
...
-					String toolIconPath = "background-image: url(" + Locator.getFacade().getEventRegistryService().getToolIcon(toolId) + ");";
-					String style = "background-position:left center; background-repeat:no-repeat; margin-left:3px; padding-left:20px; "+toolIconPath;
-					return style;
-				}
-				return null;
-			}		
+				String style = "display:block;";
+				return style;
+			}
+			public String getClass(Object object) {
```
Comment (ern) [inline sitestats/sitestats-tool/src/java/org/sakaiproject/sitestats/tool/wicket/pages/ReportsEditPage.java:922]: the use of method names getClass should probably be avoided

Comment (jesusmmp) [reply sitestats/sitestats-tool/src/java/org/sakaiproject/sitestats/tool/wicket/pages/ReportsEditPage.java]: Changed to getIconClass.

**Review by ern — DISMISSED** [review]

File: sitestats/sitestats-tool/src/java/org/sakaiproject/sitestats/tool/wicket/components/StylableSelectOptionsGroup.java [hunk 2]
```diff
@@ -26,13 +26,16 @@
 public class StylableSelectOptionsGroup extends Border {
 	private static final long	serialVersionUID	= 1L;
 
-	public StylableSelectOptionsGroup(String id, IModel model, IModel style) {
+	public StylableSelectOptionsGroup(String id, IModel model, IModel style, IModel hclass) {
```
Comment (plukasew) [inline sitestats/sitestats-tool/src/java/org/sakaiproject/sitestats/tool/wicket/components/StylableSelectOptionsGroup.java:29]: Use IModel<String> so you don't have to cast it below

File: sitestats/sitestats-tool/src/webapp/html/pages/ReportsEditPage.html [hunk 3]
```diff
@@ -132,7 +132,7 @@ <h4 class="summaryHeader">
                             <td>
                                 <!-- Tool selection -->
                                 <div id="what-tools-select" style="display:none">
-	                               <select wicket:id="reportParams.whatToolIds" style="overflow: auto;" size="7" multiple="multiple" class="what-tools-select-component responsive-form">
+	                               <select wicket:id="reportParams.whatToolIds" style="overflow: auto; font-family: FontAwesome;" size="7" multiple="multiple" class="what-tools-select-component responsive-form">
```
Comment (plukasew) [inline sitestats/sitestats-tool/src/webapp/html/pages/ReportsEditPage.html:135]: Remove this inline styles and put them in an appropriate CSS class instead

**Review by bjones86 — CHANGES_REQUESTED** [review]
Changes requested as per Paul's review

File: sitestats/sitestats-tool/src/webapp/html/pages/ReportsEditPage.html (hunk 3 above)
Comment (jesusmmp) [reply sitestats/sitestats-tool/src/webapp/html/pages/ReportsEditPage.html:135]: HTML pages on Sitestats tool are really a disaster about inline CSS, please file a new JIRA for that purpose. Thanks!

**Review by ern — APPROVED** [review]

**Pull request merged**